src/_includes/v2/footer/mobile.html

```diff
@@ -0,0 +1,9 @@
+<footer class="d-flex flex-column d-lg-none footer container-fluid row-gap-3">
+  {% include v2/footer/calitp-logo.html %} {% include v2/footer/blurb.html %}
```

Comment:
In my opinion, two lines here would be a bit more readable.

```suggestion
  {% include v2/footer/calitp-logo.html %}
  {% include v2/footer/blurb.html %}
```